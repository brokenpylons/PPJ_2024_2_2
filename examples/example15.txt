var n = 5;
var sum = 0;
for (var i = 1, n) {
    sum = sum + i * i;
}

print(sum);
