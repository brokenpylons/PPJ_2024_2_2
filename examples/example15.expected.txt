real("5") integer-divides("//") real("2") 
